docs/source/usersguide/geometry.rst

```diff
@@ -434,3 +438,30 @@ named ``dagmc.h5m``) when initializing a simulation. If a `geometry.xml
   <https://svalinn.github.io/DAGMC/usersguide/tools.html#make-watertight>`_. Future
   implementations of DAGMC geometry will support small volume overlaps and
   un-merged surfaces.
+
+-------------------------
+Calculating Atoms Content
+-------------------------
+
+If the total volume occupied by all instances of a cell in the geometry is known
+by the user, it is possible to assign this volume to a cell without performing a stochastic volume
+calculation::
+
+  from uncertainties import ufloat
+
+  # Set known total volume in [cc]
+  cell = openmc.Cell()
+  cell.volume = 17.0
+
+  # Set volume if it is known with some uncertainty
+  cell.volume = ufloat(17.0, 0.1)
+
+Once a volume is set, and a cell is filled with a material or distributed
+materials, it is possible to use the :func:`~openmc.Cell.atoms` method to obtain
+a dictionary of nuclides and their total number of atoms in all instances
+of a cell (e.g. ``{'H1':1.0e22, 'O16':0.5e22, ...}``)::
```

Comment:
```suggestion
of a cell (e.g. ``{'H1': 1.0e22, 'O16': 0.5e22, ...}``)::
```

Reply:
I applied the change in Commit 2debfc0d8a56e969c76af97ac377232a2d69e744.